docs/api-types/index-v1alpha1.md

```diff
@@ -282,7 +283,7 @@ Policy specifies a policy to use for Attestation or the CIP validation (iff at l
 | ----- | ----------- | ------ | -------- |
 | type | Which kind of policy this is, currently only rego or cue are supported. Furthermore, only cue is tested :) | string | true |
 | data | Data contains the policy definition. | string | false |
-| url | URL to the policy data. | apis.URL | false |
+| remote | Remote defines the url to a policy. | [URL](#url) | false |
```

Comment:
Is this right? Seems like somewhere there should be the shasum field also?

Reply:
As in, shouldn't this be a pointer to #remote?

Reply:
fixed